doc/user_guide/color_usage.rst

```diff
@@ -0,0 +1,1525 @@
+                stroke=stroke_gs,               
+
+            )
+        )
+        return grayscale
+
+    def plot_scheme(scheme_name, dict_schemes, continuous=False, cvd=False, grayscale=False, top=False, bottom=False, single=True, method='rgb'):
+        # determine chart settings
+        chart_dict = chart_settings(scheme_name, dict_schemes, continuous)    
+
+        # unpack chart_dict
+        custom = chart_dict['custom']
+        data = chart_dict['data']
+        w = chart_dict['width']
+        h_l = 50
+        h_s = 20
+        cr_l = chart_dict['cornerRadius_large']
+        cr_s = chart_dict['cornerRadius_small']
+        no = chart_dict['no_colors']
+
+        # define color scale and stroke
+        color_scale = chart_color_scale(scheme_name, dict_schemes, continuous, custom, no, method)
+
+        # define how hex-codes can be derived, from datasource or using color scale
+        if single:
+            expr_hex = "datum.hex" if custom and not continuous else "scale('color', datum.i)"
+        elif top:
+            expr_hex = "datum.hex" if custom and not continuous else "scale('concat_0_color', datum.i)"
+        elif bottom:
+            expr_hex = "datum.hex" if custom and not continuous else "scale('concat_1_color', datum.i)"
+
+        # define styling for y-axis
+        my_y_axis = alt.Axis(
+            grid=False,
+            ticks=False,
+            labels=False,
+            domain=False,
+            titleAngle=0,
+            titleAlign="right",
+            titleBaseline="middle",
+        )    
+
+        # determine color pallettes for different color deficiencies
+        normal_vision = chart_normal_vision(data, h_l, w, expr_hex, my_y_axis, cr_l, no)
+        green_blindness = chart_green_blindness(data, h_s, w, expr_hex, my_y_axis, cr_s, no)
+        red_blindness = chart_red_blindness(data, h_s, w, expr_hex, my_y_axis, cr_s, no)
+        monochrome = chart_grayscale(data, h_s, w, expr_hex, my_y_axis, cr_s, no)
+
+        # determine which color pallettes to return
+        if cvd and grayscale:
+            chart_concat = (normal_vision & green_blindness & red_blindness & monochrome & color_scale)
+        elif cvd and not grayscale:
+            chart_concat = (normal_vision & green_blindness & red_blindness & color_scale)
+        else:
+            chart_concat = (normal_vision & color_scale)
+
+        # set configuration on concatenated chart object
+        if single:
+            chart_concat = chart_concat.configure_concat(spacing=1).configure_view(stroke=None)    
+        return chart_concat
+
+
+The basics of color selection
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Color selection is an essential aspect of creating clear and informative
+visualizations of scientific data. To make the most effective choices,
+it's important to understand the basics of color perception and theory.
+
+Color perception and theory
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Instead of starting with talking about how to define a path between two colors, what do you think about having a more data-centric explanation that explains the different scenarios for how color could be used with data? These would be varying the hue/color to represent categorical values and varying mainly luminance to represent change in a quantitative value (with two sub categories: sequential and diverging colormaps, both which should be perceptually uniform). Personally, I'm a fan of how the seaborn docs introduce this topic as it becomes really tangible how the concepts of color applies to data right across the bath. I think talking about categorical and numerical data also maps nicely to that these concepts are already used in channel encodings and maybe our categories would be `nominal` data on the one hand, and then `ordinal` and `quantitative` together (as discrete and continuous sequential (or diverging) color scales). What do you think?

The section you have is really interesting and I love that the color space plot is made directly in Altair, so I want to keep it as part of the "Sequential" subsection where we talk more about how these color palettes are defined.

Reply:
I start my class on this with these three slides:

![image](https://github.com/altair-viz/altair/assets/4560057/16345a6f-60f0-4cc4-8184-50b40d85dff1)

![image](https://github.com/altair-viz/altair/assets/4560057/c47f1c10-0b4d-4553-aeca-8efed03cea5d)

![image](https://github.com/altair-viz/altair/assets/4560057/fb98e0a5-819a-4749-bd7f-aff50d2e83f9)

Reply:
I also really like these from the mpl docs on what perceptually uniform looks like in terms of the lightness increase:

![image](https://github.com/altair-viz/altair/assets/4560057/7eac9e2f-dd09-4771-8464-09f6f498891a)

![image](https://github.com/altair-viz/altair/assets/4560057/f8aa7d81-5d89-4157-8ee6-ec84d3e14e55)